hironx_ros_bridge/rqt_plugin.xml

```diff
@@ -0,0 +1,22 @@
+<library path="src">
+  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
+    <description>
+      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
+    </description>
+    <qtgui>
+      <group>
+        <label>Robot</label>
+        <icon type="theme">folder</icon>
+        <statustip>Plugins related to specific robots.</statustip>
+      </group>
+      <group>
```

Comment:
I'm using catkin (not catkni_make) so it's not exactly same situation. Any
way how did rqt find the plugins? running external command?

--
◉ Kei Okada


On Wed, Feb 4, 2015 at 2:27 AM, Isaac I.Y. Saito <notifications@github.com>
wrote:

> In hironx_ros_bridge/rqt_plugin.xml
> <https://github.com/start-jsk/rtmros_hironx/pull/317#discussion_r24021524>
> :
>
> > @@ -0,0 +1,22 @@
> > +<library path="src">
> > +  <class name="HiroNXDashboard" type="hironx_ros_bridge.hironx_dashboard.HiroNXDashboard" base_class_type="rqt_gui_py::Plugin">
> > +    <description>
> > +      A Python GUI plugin for displaying a dashboard that displays and interacts with low level hrpsys states.
> > +    </description>
> > +    <qtgui>
> > +      <group>
> > +        <label>Robot</label>
> > +        <icon type="theme">folder</icon>
> > +        <statustip>Plugins related to specific robots.</statustip>
> > +      </group>
> > +      <group>
>
> rqt --force-discover nor --clear-config didn't work for me.
>
> Somehow QtGui couldn't find this via plugin mechanism. When I catkin_make
> installed and source from its result (instead of sourcing from devel) I
> got it to work.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/start-jsk/rtmros_hironx/pull/317/files#r24021524>.
>